xml/telnet_banners.xml

```diff
@@ -816,4 +816,589 @@
+    <example _encoding="base64" host.name="BJ-IDC-FW2">
+     SVBTTy9pMzg2IChCSi1JREMtRlcyKSAodHR5cDcpCgoKClRoaXMgc3lzdGVtIGlzIGZvciBhdXRob3Jpem
+     VkIHVzZSBvbmx5LgoKCgoKCgoKbG9naW46IA==
+    </example>
+    <param pos="0" name="hw.vendor" value="Check Point"/>
+    <param pos="0" name="hw.family" value="Check Point"/>
+    <param pos="0" name="hw.device" value="Firewall"/>
+    <param pos="0" name="hw.product" value="IPSO"/>
+    <param pos="1" name="host.name"/>
+  </fingerprint>
+  <fingerprint pattern="Tasman Networks Inc.*Telnet Login">
+    <description>Tasman Networks Login</description>
+    <example _encoding="base64" hw.vendor="Tasman Networks">
+      Iy0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0
+      tLS0tLS0tCiMgVGFzbWFuIE5ldHdvcmtzIEluYy4gVGVsbmV0IExvZ2luCiMtLS0tLS0tLS0tLS0tLS0tLS0tLS
+      0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLS0tLQpFc2NhcGUgY2hhcmFjd
+      GVyIGlzICdeXScuCgoKICAgICAgICAKbG9naW46IA==
+    </example>
+    <param pos="0" name="hw.vendor" value="Tasman Networks"/>
+    <param pos="0" name="hw.device" value="Router"/>
+    <param pos="0" name="hw.product" value="Tasman Networks router"/>
+    <param pos="0" name="hw.family" value="UNKNOWN"/>
+  </fingerprint>
+  <fingerprint pattern="Pragma Systems">
+    <description>MS Windows running Pragma TelnetD server</description>
+    <example _encoding="base64">
+      V2VsY29tZSB0byBHZW1hZGVwdCBMb2dpc3RpY3MgUkYgU2VydmVyCihDKSBDb3B5cmlnaHQgMTk5NC0yMDEyIFB
+      yYWdtYSBTeXN0ZW1zLCBJbmMuCgpsb2dpbiBuYW1lOiA=
+    </example>
+    <param pos="0" name="hw.vendor" value="Microsoft"/>
+    <param pos="0" name="hw.family" value="Windows"/>
+    <param pos="0" name="hw.device" value="General"/>
+    <param pos="0" name="hw.product" value="Windows"/>
+  </fingerprint>
+  <fingerprint pattern="Application Required. No Installation Default">
+    <description>probably IBM AS/400 running TN3270 or 5250 emulation server</description>
+    <example _encoding="base64">
+     QXBwbGljYXRpb24gUmVxdWlyZWQuIE5vIEluc3RhbGxhdGlvbiBEZWZhdWx0ICAgICAgICA
+     gICAgICAgICAgICAgICAgICAgICAgICAgIApFbnRlciBBcHBsaWNhdGlvbiBOYW1lOg==
+    </example>
+    <param pos="0" name="hw.vendor" value="IBM"/>
+    <param pos="0" name="hw.device" value="General"/>
+    <param pos="0" name="hw.product" value="UNKNOWN"/>
+    <param pos="0" name="hw.family" value="UNKNOWN"/>
+  </fingerprint>
+  <fingerprint pattern="This copy of the Ataman TCP Remote Logon Services">
+    <description>Windows NT/2k/2k3 running Ataman telnet server</description>
+    <example _encoding="base64">
+      VGhpcyBjb3B5IG9mIHRoZSBBdGFtYW4gVENQIFJlbW90ZSBMb2dvbiBTZXJ2aWNlcyBpcyByZWdpc3RlcmVkIG
+      FzIGxpY2Vuc2VkIHRvOgoJRUNJMi9ERE1TCgpBY2NvdW50IE5hbWU6IA==
+    </example>
+    <param pos="0" name="hw.vendor" value="Microsoft"/>
+    <param pos="0" name="hw.family" value="Windows"/>
+    <param pos="0" name="hw.device" value="General"/>
```

Comment:
Here and everywhere, `hw.device` of `General` should be avoided.  It provides no value.